quic/api/QuicTransportBase.cpp

```diff
@@ -1636,10 +1636,19 @@ QuicTransportBase::notifyPendingWriteOnStream(StreamId id, WriteCallback* wcb) {
   if (!qStream->writable()) {
     return folly::makeUnexpected(LocalErrorCode::STREAM_CLOSED);
   }
+
+  if (wcb == nullptr) {
```

Comment:
why do you make this change?

Reply:
Is it not necessary? Otherwise the deref on L1653 would fail

Reply:
ye i guess this is fine. I was under the impression that today someone can pass in a nullptr as write callback to uninstall write callback for a stream. After reading the code i realize it's not that case at all.